test-clients/src/main/java/com/hedera/services/bdd/suites/contract/opcodes/CreateSuite.java

```diff
@@ -264,6 +280,116 @@ HapiApiSpec resetOnStackedFactoryFailureWorks() {
 				);
 	}
 
+	private HapiApiSpec contractCreateWithNewOpInConstructor(
+			final String testName,
+			final String filePath,
+			final long expectedChildrenCount) {
+		final String bc = testName + "Bytecode";
+		final String txn = testName + "Txn";
+		final String parent = testName + "ParentInfo";
+
+		return defaultHapiSpec("ContractCreateWithNewOpInConstructor-" + testName)
+				.given(
+						fileCreate(bc).path(filePath)
+				).when().then(
+						contractCreate(testName).bytecode(bc).via(txn),
+						getContractInfo(testName).saveToRegistry(parent).logged(),
+						getTxnRecord(txn)
+								.saveCreatedContractListToRegistry(testName)
+								.logged(),
+						UtilVerbs.contractListWithPropertiesInheritedFrom(testName + "CreateResult", expectedChildrenCount, parent)
+				);
+	}
+
+	HapiApiSpec contractCreateContractSpec() {
```

Comment:
It seems that this test:
- creates a child contract
- sets storage to the child contract
In order to make it more clear, we can rename the test to `childContractStorageWorks`. We can simplify the code a lot by:
- removing all the hardcoded `balances`, `gas` etc
- remove `cryptoCreate` operation
- assert the return values of contract local call by doing something similar to `GlobalPropertiesSuite`
- Remove the hardcoded `ContractResources.CREATED_TRIVIAL_CONTRACT_RETURNS` value and declare it here. I think that it will be better compared to setting it in Contract Resources